src/test/java/org/prebid/server/auction/AmpRequestFactoryTest.java

```diff
@@ -361,6 +343,654 @@ public void shouldRespondWithBidRequestWithTestFlagOn() {
         assertThat(captor.getValue().getTest()).isEqualTo(1);
     }
 
+    @Test
+    public void shouldReturnBidRequestWithOverriddenTagIdBySlotParamValue() {
+        // given
+        given(httpRequest.getParam("slot")).willReturn("Overridden-tagId");
+        given(storedRequestProcessor.processAmpRequest(anyString()))
+                .willReturn(Future.succeededFuture(givenBidRequestWithExt(null,null)));
+        given(auctionRequestFactory.fillImplicitParameters(any(), eq(routingContext)))
+                .willAnswer(answerWithFirstArgument());
+        given(auctionRequestFactory.validateRequest(any())).willAnswer(answerWithFirstArgument());
+
+        // when
+        final Future<BidRequest> future = factory.fromRequest(routingContext);
+
+        // then
+        assertThat(future.succeeded()).isTrue();
+
```

Comment:
There's no need for extra space between these lines, same applies to other tests below as well